### PR TITLE
fix(textarea): set cursor attribute to text when textarea is readonly

### PR DESCRIPTION
**Pull request by chenc041**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [x] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- Fix: 修复 TextArea readonly 模式下光标显示为禁用问题 [@chenc041](https://github.com/chenc041)  [#535](https://github.com/DouyinFE/semi-design/issues/535)

---

🇺🇸 English
- Fix: Fixed TextArea readonly hover cursor style bug [@chenc041](https://github.com/chenc041)  [#535](https://github.com/DouyinFE/semi-design/issues/535)

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### Other
- [ ] Skip Changelog

### CODE REVIEW

File: packages/semi-foundation/input/textarea.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@ $module: #{$prefix}-input;
         cursor: default;
     }
 
-    &-disabled {
+    &-disabled, &-readonly {
```
Comment (shijiatongxue) [inline packages/semi-foundation/input/textarea.scss:62]: 58 ~ 60 行可以干掉，下面覆盖了上面的样式

Comment (shijiatongxue) [reply packages/semi-foundation/input/textarea.scss:62]: image

Comment (chenc041) [reply packages/semi-foundation/input/textarea.scss:62]: 好的 稍等会

Comment (shijiatongxue) [reply packages/semi-foundation/input/textarea.scss:58]: 可以更新下哈

Comment (chenc041) [reply packages/semi-foundation/input/textarea.scss:58]: 已经更新好了呀 你看看呢

**Comment by shijiatongxue** [comment]
2.4.0 fixed

**Pull request merged**